domain-proxy/pkg/server/server.go

```diff
@@ -0,0 +1,246 @@
+package server
+
+import (
+	"bufio"
+	"fmt"
+	. "github.com/redhat-appstudio/jvm-build-service/domain-proxy/pkg/common"
+	"net"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	HttpPort                     = 80
+	HttpsPort                    = 443
+	ProxyTargetWhitelistKey      = "PROXY_TARGET_WHITELIST"
+	DefaultProxyTargetWhitelist  = "gariscus.com,neverssl.com,repo1.maven.org,repo.maven.apache.org,repository.jboss.org,packages.confluent.io,jitpack.io,repo.gradle.org,plugins.gradle.org"
```

Comment:
What are the first two?

Reply:
Those shouldn't have been committed. They were part of a separate changelist I had for testing but then got committed when I was doing refactoring. They'll be removed.

Reply:
They've been removed.